### ISSUE TITLE
[MSCrashes] add option to have crash reporting be sync instead of async

### ISSUE THREAD

**Issue by lwouis** [microsoft/appcenter#1954]
**Describe the solution you'd like**

I would like to be able to have a sync'd step when my app starts, so that I can check for a crash in previous session, and if there is, do the crash report UX with the user, and **only after** this is done (which is where I need this sync option), continue loading my app.

This is required for me to cover this important use-case: the app crashes early on launch, and further restart of the app continue crashing at the same spot. That scenario today, because of MSCrashes being async, means it doesn't trigger, and the crash report UX is never shown to the user, thus I have no clue in AppCenter portal that the users are experiencing the issue. I know of these because some tech-savy directly opened issues on Github explaining the situation.

**Describe alternatives you've considered**

Another option would be to have a separate process for crash reporting. This would be more potent and flexible. However, I think a sync/async option would be easier for you guys to implement.

**Additional context**

I originally filed this as a bug on the SDK side, and they told me to open a feature request here. Here is the [original ticket with more context and discussions](https://github.com/microsoft/appcenter-sdk-apple/issues/2146#issuecomment-671905080).

**Comment by lwouis** [comment]
Hi,

It's been... a few months 😅 Could we get an update from a maintainer?

Thank you!

**Comment by MatkovIvan** [comment]
It's based on public roadmap, see details [here](https://github.com/Microsoft/appcenter/wiki/Roadmap)

**Comment by lwouis** [comment]
This issue was closed without explanation. Could someone please explain what happened?

**Comment by akamenev** [comment]
Thank you for the feature request @lwouis! Currently, we are focusing on improving the reliability and performance of the service and new feature work is significantly reduced. To keep the team focused we are closing feature requests with no activity in the past 30-60 days. If you believe this is still relevant, feel free to create a new issue and tag this item so that we can review it. Sorry for the inconvenience.
